lib/organisms/gravity-form/gravity-form.controller.js

```diff
@@ -1,11 +1,169 @@
-angular
-    .module('lnPatterns')
-    .controller('lnOGravityFormController', lnOGravityFormController);
+(function () {
```

Comment:
Cool, that's only there because I started building it in a super small footprint project seed I use, will sort it out for next pull request